pyproject.toml

```diff
@@ -100,14 +109,17 @@ select = [
     # flake8-simplify
     "SIM",
     # isort
-    # "I",
+    "I",
     "G",
 ]
 ignore = [
     # star imports
     "F405", "F403",
     # lambda expression assignment
     "E731",
+    # Ignore line to long. Most cases are either Exception
+    # or inline notes/docstring
+    "E501",
```

Comment:
Instead of doing this we should adopt the `black` line length of 88. Currently this file sets it to `80`